### DISCUSSION TITLE
Provisioning the theme  color in the Dockerfile of a code server image

### DISCUSSION THREAD

**Discussion by BouarfaMahi** [coder/code-server#6377] [General]
how can i do that?
thanks

**Comment by code-asher** [comment]
Normally you could do this by editing the user settings JSON file 
but I think for VS Code web the theme is saved in browser storage 
so I am not sure this is possible, unfortunately.

**Reply by BouarfaMahi** [reply]
Thank you for your valuable advice. I will try.
